ats/src/ats/machines.py

```diff
@@ -571,13 +573,13 @@ def _launch(self, test):
                 if configuration.options.sleepBeforeSrun > 0:
                     if MachineCore.printSleepBeforeSrunNotice:
                         MachineCore.printSleepBeforeSrunNotice = False
-                        print "ATS Info: MachineCore._launch Will sleep %d seconds before each srun " % configuration.options.sleepBeforeSrun
+                        print("ATS Info: MachineCore._launch Will sleep %d seconds before each srun " % configuration.options.sleepBeforeSrun)
```

Comment:
"ATS Info:" is inconsistent with the previous "INFO:"

Reply:
Indeed it is. The goal of this PR is simply to inch closer to the Python 3 upgrade.

Reply:
Perfectly acceptable response =) there is a generally a lot of inconsistency in message type prefixes that are out of the scope of this PR.  Thanks for for working towards this!